documentation/modules/ROOT/pages/configuration/signalling.adoc

```diff
@@ -243,6 +244,47 @@ endif::community[]
 * xref:{link-postgresql-connector}#postgresql-ad-hoc-snapshots[PostgreSQL connector ad hoc snapshots]
 * xref:{link-sqlserver-connector}#sqlserver-ad-hoc-snapshots[SQL Server connector ad hoc snapshots]
 
+[id="debezium-signaling-stop-ad-hoc-snapshots]
+=== Ad hoc snapshot stop signals
+
+You can request a connector to stop an in-progress ad hoc snapshot by creating a signal table entry with the `stop-snapshot` signal type.
+After processing the signal, the connector will stop the current in-progress snapshot operation.
+
+You can stop ad hoc snapshots for the following {prodname} connectors:
+
+* Db2
+ifdef::community[]
+* MongoDB
+endif::community[]
+* MySQL
+ifdef::community[]
+* Oracle
+endif::community[]
+* PostgreSQL
+* SQL Server
+
+[id="debezium-signaling-example-of-a-stop-ad-hoc-signal-record"]
+.Example of a stop ad hoc snapshot signal record
+[cols="1,9",options="header"]
+|===
+|Column | Value
+
+|id
+|`d139b9b7-7777-4547-917d-e1775ea61d41`
+
+|type
+|`stop-snapshot`
+
+|data
+|`{"type":"INCREMENTAL", "data-collections": ["public.MyFirstTable"]}`
+
+|===
+
+You must specify the `type` of the signal.
+The `data-collections` field is optional.
+Leave the `data-collections` field blank to request the connector to stop all activity in the current snapshot.
+If you want the incremental snapshot to proceed, but you want to exclude specific collections from the snapshot, provide a comma-separated list of the names of the collections to exclude.
```

Comment:
@Naros The code now supports regexes, not only list. Please feel free to merge both PRs yourself when the doc is updated.